examples/account-federation-step-function/steps/add-to-close-io/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import type { Args } from "./types";
+import fetch from "node-fetch";
+
+export async function run({ event }: Args) {
+
+  const auth = btoa(`${process.env.CLOSE_API_KEY}:`);
```

Comment:
Ooh base64ing the env vars! Does this need documenting?

Reply:
No, this is a close specific basic auth need :)